### ISSUE TITLE
req: reduce size of height for buttons, not fit in minilaptops

### ISSUE THREAD

**Issue by mckaygerhard** [LordShenron/Project-Ignite#1]
hi @LordShenron this its a great theme, due despiste the dark, the items are separated correctly

but for small screens and minilaptops the buttons are too big only by the height size..

can be possible reduce the height or tell me how to do so!?

**Comment by LordShenron** [comment]
Yes, it is possible, you need to edit the gtk files according to your need. Since, mine is a 15.6" device, i made this according to that.

**Comment by mckaygerhard** [comment]
some point where i must edit the files? (to cheap time)

**Comment by LordShenron** [comment]
You have to edit all the files that are specific to the DE you are using.

**Comment by mckaygerhard** [comment]
ka ka ka je je, sorry for my bad english, i mean mostly aprox what part of the rc file (gtk2) or css file (gtk3) of the directoryes¡? i'll modify all due in minilaptops for distribution users will use desktops

**Comment by LordShenron** [comment]
you have to edit main rc in gtk2 directory, css in gtk3 directories and you have to modify metacity window managers and gnome shell too.
